plutus-core/plutus-core/src/PlutusCore/Builtins.hs

```diff
@@ -106,6 +109,8 @@ instance Pretty DefaultFun where
     pretty IfThenElse           = "ifThenElse"
     pretty CharToString         = "charToString"
     pretty Append               = "append"
+    pretty EncodeUtf8           = "encodeUtf8"
+    pretty DecodeUtf8           = "decodeUtf8"
```

Comment:
Tangential: @michaelpj I'm so annoyed that we don't derive `Pretty` in terms of `Show` for `DefaultFun`, because of stuff like

```haskell
pretty GreaterThanEqInteger = "greaterThanEqualsInteger"
```

Adding that `pretty EncodeUtf8 = "encodeUtf8"` line (and the other one) manually is stupid redundant work. Should we rename the constructors of `DefaultFun` to match with the `Pretty` instance or should we change the `Pretty` instance to be derived in terms of `Show` (with the first letter lowered) even if that breaks backwards compatibility?

Reply:
Backwards compatibility for whom? Nobody uses the parser. I say change it. Not in this PR, though.